vagrant-pxe-airgap-harvester/README.md

```diff
@@ -0,0 +1,119 @@
+Harvester iPXE Boot Using Vagrant Libvirt
+=========================================
+
+Introduction
+------------
+
+**Note:** if you do not desire to have an environment air-gapped it is strongly-encouraged to simply run the `vagrant-pxe-harvester` ipxe-example over this one - as this one is not *only* **more** resource intensive but will take a **substantially** long time to provision and *currently* has some known issues.
+
+**Also Note:** this set up with the `settings.yml` is currently intended to run with `rancher_config.run_single_node_rancher`, `rancher_config.run_single_node_air_gapped_rancher`, and `harvester_network_config.offline` set to `true` - there are known issues with the setup not working if those values are changed.
+
+Utilizing [Vagrant][vagrant], [KVM][kvm], and [Ansible][ansible] to create a
+ready-to-play virtual Harvester & Rancher environment for evaluation and testing
+purposes. Two Vagrant VMs will be created by default, PXE server and a
+single or multi-node Harvester respectively - with the optional vm for the Rancher instance with baked in Docker Registry, K3s & Helm installed Rancher.
+
+Known Issues
```

Comment:
Can we move `Known Issues` & `troubleshooting` at the end of the Readme?

Reply:
sounds good :smile: -updated